### PR TITLE
new class for test

### PR DESCRIPTION
**Pull request by Devaraj-Shirur**
I have sent you a new update of locators

### CODE REVIEW

File: Collebra/src/locators/Hello.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package locators;
+
+public class Hello {
+
+	public static void main(String[] args) {
+		System.out.println("hello le navvya");
+	}
+
```
Comment (naveen5257) [inline Collebra/src/locators/Hello.java:8]: where is the script

Comment (Devaraj-Shirur) [reply Collebra/src/locators/Hello.java:8]: ok sir i will edit as soon as possible

Comment (naveen5257) [reply Collebra/src/locators/Hello.java:8]: its ok do if fast

Comment (Devaraj-Shirur) [reply Collebra/src/locators/Hello.java:8]: ok sir thank you

Comment (naveen5257) [reply Collebra/src/locators/Hello.java:8]: other wise u won't get salary of this month

**Pull request merged**